Checkpoint3/Checkpoint3.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Nice job, your control flow between the different moving parts could use a little work, but nice job.